Update feature list for CSS color module (#38286)
mdn/content: files/en-us/web/css/css_colors/index.md

### At-rules and descriptors

> [!NOTE]

> > The CSS color module introduces the {{cssxref("@color-profile")}} at-rule that have not been implemented, along with its descriptors: `components`, `rendering-intent` and `src`.

Reviewer: ```suggestion
> The CSS color module introduces the {{cssxref("@color-profile")}} at-rule, along with its `components`, `rendering-intent` and `src` descriptors. These features have not yet been implemented in any browser.
```

  - [`oklch()`](/en-US/docs/Web/CSS/color_value/oklch)
  - [`color()`](/en-US/docs/Web/CSS/color_value/color)
- [`color-mix()`](/en-US/docs/Web/CSS/color_value/color-mix)
- {{CSSXref("color_value/light-dark", "light-dark()")}}

> [!NOTE]

> > The CSS color module introduces the {{CSSXref("color_value/device-cmyk", "device-cmyk()")}} and the `contrast-color(),` functions that have not been implemented.

Reviewer: ```suggestion
> The CSS color module introduces the {{CSSXref("color_value/device-cmyk", "device-cmyk()")}} and `contrast-color()` functions, which have not yet been implemented in any browser.
```

### Interfaces

> [!NOTE]

> > The CSS color module introduces the `CSSColorProfileRule` interface that have not been implemented.

Reviewer: ```suggestion
> The CSS color module introduces the `CSSColorProfileRule` interface, which has not been implemented in any browser.
```
